nexus_constructor/validators.py

```diff
@@ -378,3 +378,29 @@ def validate(self, input: str, pos: int) -> QValidator.State:
         return QValidator.Acceptable
 
     is_valid = Signal(bool)
+
+
+class HDFValidator(QValidator):
```

Comment:
I think this would benefit from a more specific name, maybe `HDFLocationExistsValidator`?